data/adapters/adastra_asb_adapter.py

```diff
@@ -1,96 +1,160 @@
 import csv
+import json
+import os
+
 from adapters import Adapter
+from db.arango_db import ArangoDB
 from adapters.helpers import build_variant_id
 
-# Example ADASTRA allele-specific binding (ASB) file TF_name@cell_name.tsv (e.g. ATF1_HUMAN@HepG2__hepatoblastoma_.tsv):
+# ADASTRA allele-specific binding (ASB) file downloaded from: https://adastra.autosome.org/assets/cltfdata/adastra.cltf.bill_cipher.zip
+# Cell ontology available from GTRD (Gene Transcription Regulation Database): http://gtrd.biouml.org/
+
+# Example file TF_name@cell_name.tsv (e.g. ATF1_HUMAN@HepG2__hepatoblastoma_.tsv):
 # chr	pos	ID	ref	alt	repeat_type	mean_BAD	mean_SNP_per_segment	total_cover	n_aggregated	es_mean_ref	es_mean_altlogitp_ref	fdrp_bh_ref	logitp_alt	fdrp_bh_alt	motif_log_pref	motif_log_palt	motif_fc	motif_pos	motif_orient	motif_conc	novel
 # chr11	129321262.0 rs10750410	A	G		1.25	518.5	73.0	2.0	-1.508200583122832	1.5220631227173078	0.9999438590195968	1.0	3.776801544248756e-06	0.0024711390339222	2.668977799202377	2.9239362481887974	0.8469536347168959	19	+	No Hit	False
 
 
 class ASB(Adapter):
     # 1-based coordinate system
+    ALLOWED_LABELS = ['asb', 'asb_cell_ontology']
+    ONTOLOGY_PRIORITY_LIST = ['CL:', 'UBERON:', 'CLO:', 'EFO:']
+    CELL_ONTOLOGY_ID_MAPPING_PATH = './samples/allele_specific_binding/ADASTRA_cell_ontologies_mapped_ids.tsv'
```

Comment:
we should move these files to a different folder if they are necessary to load production data. I talked to Mingjie and agreed to create a folder called `data_loading_support_files`